org-formation/600-access/_tasks.yaml

```diff
@@ -214,6 +214,21 @@ StackArmorReadOnlyAccess:
       - arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess
       - arn:aws:iam::aws:policy/AmazonVPCReadOnlyAccess
 
+# Setup SynapseProd cross-account access the BedRock in SynapseLlmProd
+# https://repost.aws/knowledge-center/bedrock-invoke-with-cross-account
+SynapseProdBedrockFullAccess:
```

Comment:
shouldn't this be `SynapseLlmProdFullAccess` since you are giving access to Synapse LLM?

Reply:
I viewed it as giving SynapseProd full access to Bedrock (in the SynapseLLM account). With the comment, I think it's OK either way.